### PR TITLE
Blocking import for gson as we embed it

### PR DESCRIPTION
**Pull request by cschneider**

### CODE REVIEW

File: zipkin/pom.xml [hunk 1]
```diff
@@ -63,6 +63,7 @@
 
   <build>
     <plugins>
+      <!-- Need to block import of shaded packages as maven bundle plugin analyzes the unshaded jar -->
```
Comment (codefromthecrypt) [inline zipkin/pom.xml]: maybe remind people the file that blocked imports go is `bnd.bnd`? I think many don't know this

Comment (cschneider) [reply zipkin/pom.xml]: Makes sense .. I also found another issue

**Comment by cschneider** [comment]
I am now at the stage where I can install zipkin and zipkin reporter bundles. I will now try to first get everything installed and started in OSGi and then write a first actual example that uses it.

**Comment by codefromthecrypt** [comment]
cool beans.. all of the snapshot repos should be working, so please ping back how it goes.

**Comment by codefromthecrypt** [comment]
fyi this (and other related change) is going out as 1.14.4

**Pull request merged**